src/EbmlElement.cpp

```diff
@@ -518,7 +518,7 @@ filepos_t EbmlElement::RenderHead(IOCallback & output, bool bForceRender, Should
   if (EBML_ID_LENGTH((const EbmlId&)*this) <= 0 || EBML_ID_LENGTH((const EbmlId&)*this) > 4)
     return 0;
 
-  UpdateSize(writeFilter, bForceRender);
+  UpdateSize(std::move(writeFilter), bForceRender);
```

Comment:
I'm not familiar with std::move(). But looking at the [doc](https://en.cppreference.com/w/cpp/utility/move) it means transfering the ownership. This is not the case here. In fact the parameter should probably be passed by reference.

Reply:
Somewhat.

std::move is used to avoid copying. I’m intentionally avoiding breaking API.

Reply:
There's no need to preserve the API, this a new API in 2.0